src/Controllers/ContentController.php

```diff
@@ -1537,16 +1561,25 @@ private function attachInstructorLessons(mixed $content, Request $request): mixe
         }
 
         $includedFields = [];
-        $includedFields[] = 'instructor,' . $content['id'];
+        $includedFields[] = 'instructor,'.$content['id'];
 
         $requiredFields = $request->get('required_fields', []);
         $includedFields = array_merge($request->get('included_fields', []), $includedFields);
         $requiredUserState = $request->get('required_user_states', []);
         $includedUserState = $request->get('included_user_states', []);
 
+        if($request->has('title')){
+            $requiredFields = array_merge($requiredFields, ['title,%'.$request->get('title').'%,string,like']);
+        }
+
+        if ($request->has('count_filter_items')) {
+            ContentRepository::$countFilterOptionItems = $request->has('count_filter_items');
+        }
```

Comment:
Just want to confirm this is the desired outcome: that if the `count_filter_items` is on the request with any value, that the ContentRepository::$countFilterOptionItems will be set to true (even if it's set to false, 0, "", etc).

Reply:
It's important only to have the count_filter_items on the request; it was inserted only for the filters v2. In the old builds the parameter not exists